kolibri_explore_plugin/assets/src/modules/topicsRoot/handlers.js

```diff
@@ -1,6 +1,7 @@
 import { ContentNodeResource } from 'kolibri.resources';
 import urls from 'kolibri.urls';
-import { PageNames, CustomChannelApps } from '../../constants';
+import { PageNames } from '../../constants';
+import { CustomChannelApps } from '../../CustomApps';
```

Comment:
:+1: makes sense to have a separate module. Nit: the module file name should not be capitalized to be consistent with the rest:
```suggestion
import { CustomChannelApps } from '../../customApps';
```